.gitignore

```diff
@@ -80,4 +80,7 @@ docs/api
 
 # Rider
 .idea/
-.idea_modules/
\ No newline at end of file
+.idea_modules/
+
+# Misc project metadata
+.specs/
```

Comment:
Out of curiosity, what's this for? If it's some sort of dev tool, it might be better to call out what it is.